.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "common"]
```

Comment:
Do we need to use submodules? We haven't started using them elsewhere in openshift and I don't want to start here if possible.

Reply:
python, ruby and other languages in sclorg project (these are used in OpenShift officially) have already started using it.
e.g.
https://github.com/sclorg/s2i-python-container
https://github.com/sclorg/s2i-ruby-container

So, I think that s2i in this project also should start using it.

Reply:
I was thinking of the openshift github org and not the scl repos. For one, I am planning on adding this repo under prow (openshift CI) and I want to avoid submodules before we start using them elsewhere in the openshift org.

Reply:
OK, I understand. I reverted to hack/{build.sh,common.mk]. However, can you allow to use docker-squash command rather than `docker-scripts`? `docker-scripts` (https://github.com/goldmann/docker-squash.git) has already renamed to `docker-squash` and old `docker-scripts` does not work on some env.

( I mean this code https://github.com/openshift-s2i/s2i-rust/pull/3/files#diff-0631729395967455d8a8df8bc6d0eb93L53 )

Reply:
SGTM

Let me know if there is anything else that's needed here, otherwise I'll merge.

Reply:
Thank you. No problem. I would appreciate it if you could merge this PR.